src/api/posts/src/routes/feeds.js

```diff
@@ -34,6 +35,18 @@ feeds.get('/', async (req, res, next) => {
   );
 });
 
+// This should return number of jobs in the queue, active, failed, delayed, waiting jobs
+feeds.get('/info', async (req, res, next) => {
+  let jobNum;
+  try {
+    jobNum = await queue.count();
```

Comment:
Tested on GitPod and this route never seems to return (e.g., I get no response from the server, it just loads forever).  It looks like you're doing it correctly, but there's nothing coming back.